build/Mstc/MSTC.Web/Controllers/EventController.cs

```diff
@@ -29,7 +29,7 @@ public EventController()
 		}
 
 		[HttpGet]
-		public IEnumerable<EventType> BookableEvents(bool futureEventsOnly, bool withSlotsOnly, bool isAdmin = false)
+		public IEnumerable<EventType> BookableEvents(bool futureEventsOnly, bool withSlotsOnly, bool isAdmin = false, int limitBooking = 2)

```

Comment:
This method is also called from the event-admin.js script. I don't think you would want to limit the bookings for that query so I think would bebetter to make the limitBooking parameter optional here. It should work OK then as you already made it optional on the repository method :)

Reply:
agreed!

Reply:
I've updated the controller and also removed the limit in the API call